### DISCUSSION TITLE
SERVER_TOKEN??

### DISCUSSION THREAD

**Discussion by secondmanveran** [directus-labs/agency-os#122] [Q&A]
Can someone tell me what this is?

> DIRECTUS_SERVER_TOKEN="your_directus_server_token_for_server_only_routes"

I don't see it anywhere but in the readme on the repo, not in the docs anywhere, looked althrough the backend templates. 

I haven't seen much talk about it so clearly I've missed something.

Thanks

**Comment by MansourAbdi** [comment]
https://www.youtube.com/watch?v=xK4JOorrbxo&list=PLD--x9rY3ZL1tPNZxCTE_-IsFTrFGKHH-&index=3&t=300s
